draft-ietf-quic-recovery.md

```diff
@@ -1183,14 +1142,14 @@ are detected lost.
      CongestionEvent(largest_lost_packet.time)
 ~~~
 
-### On Retransmission Timeout Verified
+### On Probe Timeout Verified
 
-QUIC decreases the congestion window to the minimum value once the
-retransmission timeout has been verified and removes any packets
-sent before the newly acknowledged RTO packet.
+QUIC decreases the congestion window to the minimum value once the probe timeout
+has been verified and removes any packets sent before the newly acknowledged
+packet.
 
 ~~~
-   OnRetransmissionTimeoutVerified(packet_number)
+   OnProbeTimeoutVerified(packet_number)
```

Comment:
This looks like we will set the probetimeout verified even if this happens

    1, 2, 3 [lost], 4 [probe], 5 [probe]
    -> [ack 4]
    -> cwnd = mincwnd

in the first probe case it seems like the probe would be aggressive so it would be bad to drop the cwnd down.

even if it not aggressive right now, in the future I think we are going to tune timeouts constant per application (realtime vs interactive). This means that if we tune timeout constants too aggressively, it will result in having this reaction.

Reply:
I'd suggest we keep a "you can not do the CWND reduction on the first or 2nd PTO" to keep behavior more similar to what QUIC has now.